Source/DayViewState.swift

```diff
@@ -6,6 +6,7 @@ public protocol DayViewStateUpdating: AnyObject {
 
 public final class DayViewState {
   public private(set) var calendar: Calendar
+//    willSet {} // TODO: I think this needs to be implemented, cause timezone can change without creating a new state object
```

Comment:
I suggest ensuring that changing the timezone will always create a new state object